packages/util/src/account.ts

```diff
@@ -214,8 +213,12 @@ export const generateAddress2 = function (from: Buffer, salt: Buffer, initCode:
   assertIsBuffer(salt)
   assertIsBuffer(initCode)
 
-  assert(from.length === 20)
-  assert(salt.length === 32)
+  if (from.length !== 20) {
+    throw new Error('Expected `from` to be of length `20`.')
```

Comment:
Hi there, sorry, I know it's - viewed in isolation - an improvement, but we just have no usages of marking key words with "\`" in any other error messages in the library (and the others as well), and things would get inconsistent if we add just for these few new ones. Could you please remove?

Also please do error messages with no dots.

So, example: 'Expected `from` to be of length `20`.' -> 'Expected from to be of length 20'

For the codecov failure I guess that's ok in this case, since semantically this is not really adding functionality, it's just an - somewhat accidental - line expansion here. 🙂

Reply:
how about `Expected 'from' to be of length 20`, otherwise in some cases it might be confusing what the property name is and blends too much into the message, so i think some punctuation does help

even better if we want to add more context for helpful debugging downstream:

`Expected 'from' to be of length 20, received length ${from.length}`

Reply:
I find all of this better than before, I am just not a fan of introducing selected formatting changes, this makes things - somewhat - inconsistent. Same argument would be true for - I would guess - rather hundreds than dozens of error messages in the monorepo.

So we should rather decide to update them all or not but not starting to switch off from the common pattern on 4-5 selected ones.